### ISSUE TITLE
[FR] Upgrade Distiller to torchvision 0.3

### ISSUE THREAD

**Issue by barrh** [NervanaSystems/distiller#273]
torchvision v0.3 adds pretrained models of Resnext50 and Shufflenet v2, among others.
It requires Pytorch 1.1 or higher.
